evaluator/bundle_test.go

```diff
@@ -47,7 +47,13 @@ func TestBundleTestSuite(t *testing.T) {
 }
 
 func (s *BundleTestSuite) TestCreateServer() {
-	_, err := StartServer(context.Background(), config.DefaultConfig)
```

Comment:
nit: We can create a new `config.DefaultVanillaConfig` seems to be useful enough

Reply:
No need to pay extra attention to the bundle tests, I have another PR that deletes the entire bundle logic :)